src/wallet/rpcwallet.cpp

```diff
@@ -2620,6 +2634,25 @@ static UniValue createwallet(const JSONRPCRequest& request)
 
     wallet->postInitProcess();
 
+    // Encrypt the wallet if there's a passphrase
```

Comment:
Is there a reason to have this after the above `AddWallet`?

Reply:
Done